Sources/XcodesKit/Models.swift

```diff
@@ -96,3 +96,118 @@ public struct VersionPlist: Decodable {
     }
 }
 
+struct DownloadableRuntimesResponse: Decodable {
```

Comment:
I know every single other model is in the file - but can we move the newly added to a new file called `Runtime`